glommio/Cargo.toml

```diff
@@ -34,6 +34,7 @@ pin-project-lite = "0.1.10"
 
 [dev-dependencies]
 futures = "0.3.5"
+rand = "0.7.3"
```

Comment:
FWIW, I prefer `fast_rand` or `oorandom` crates for tests -- fewer deps, faster compile times. OTOH, we probably already have `rand` somewhere in the dep graph?

Reply:
Sure, I will check or change the crate. Not a problem at all :-)